user-manual/appraisal/appraisal.rst

```diff
@@ -290,27 +296,47 @@ edited or deleted and further acts can be added.
-resources by dragging files from the backlog to digital object components.
+Transfer files in backlog can easily be associated with ArchivesSpace resources by dragging and dropping files from the backlog to digital object components.
+
+To add files to ArchivesSpace resources:
+
+1. Search the backlog by entering terms and parameters in the search box at the top of the page.
+   
+   For a list of all transfers in backlog, simply click within the search box leaving the edit field blank, and then click **Search transfer backlog**.
+
+2. Click **ArchivesSpace** to enable the ArchivesSpace panel, :ref:`search by title or identifier of a resource <searching-ArchivesSpace-resources>`, and then click **Search ArchivesSpace**.
+   
+   A hierarchical view of the resource is displayed.
+   
+3. Create digital object components within the ArchivesSpace panel by clicking **Add new digital object**.   
+   
+   Digital object components must be created in the Appraisal tab.
+
+.. note::
+
+   Digital objects created and linked to resources in ArchivesSpace will not appear in the Appraisal tab.
+
+4. Now click and drag files in the Backlog panel to digital object components in the ArchivesSpace panel.
 
 .. image:: images/backlog_and_archivesspace.*
    :align: center
    :width: 80%
    :alt: Backlog and analysis panes
 
-Digital object components must be created in the Appraisal tab (digital objects
-created and linked to resources in ArchivesSpace will not appear in the
-Appraisal tab).
+.. note::
+
+   One file or directory from a transfer can be moved at a time. A file can only be
+   added once to an ArchivesSpace resource. Files that have been added to a digital object component will appear with a strike-through in the backlog.
+
+5. Once all files have been added and the arrangement has been set, SIPs can be
+   started in Ingest. Select the resource/archival object and click
+   **Finalize arrangement**. SIPs can be created from any level of description
+   (i.e. a SIP can be started for an entire resource or from an archival object
+   within a resource such as a series).
 
-One file or directory from a transfer can be moved at a time. A file can only be
-added once to an ArchivesSpace resource. Files that have been added to a digital
-object component will appear with a strike-through in the backlog.
+.. tip::
 
-Once all files have been added to their place in the arrangement, SIPs can be
-started in Ingest by selecting the resource/archival object and clicking
-“Finalize Arrangement.” SIPs can be created from any level of description
-(i.e. a SIP can be started for an entire resource or from an archival object
-within a resource such as a series).
+   Before starting a SIP in the ArchivesSpace panel, verify that all required metadata associated with an ArchivesSpace resource is in place, particularly at the parent-level of the arrangement. In ArchivesSpace, verify that a material description record is linked to an agent record.
```

Comment:
Question: Should 'Agent' be capitalised as a proper noun from ArchiveSpace's perspective? (I may be misunderstanding the concept of a proper-noun!)

Reply:
It looks like the ArchiveSpace team goes back and forth on this, but "agent record" is [regularly used](https://archivesspace.atlassian.net/wiki/spaces/ADC/pages/103209762/Public+Interface+Documentation+for+staff#PublicInterfaceDocumentation(forstaff)-Publishing/unpublishingAgentrecords).